js/components/volumePanel.js

```diff
@@ -0,0 +1,77 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const ControlBar = require('./controlBar');
+
+/**
+ * Volume menu. This component is used for both the Popover and
+ * Screen modes of the menu.
+ */
+class VolumePanel extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
+      <div className="oo-absolute-centered oo-volume-panel">
+        <ControlBar
```

Comment:
Sorry I hadn't noticed this before. Is it necessary to use a whole control bar for the volume button? The control bar is a fairly heavy component so it might be more efficient and a bit simpler to use the volume component on its own

Reply:
There are styles involving the control bar and calculations done in the control bar component for the volume controls. This component is also likely to be deprecated soon as the volume bar requirement has changed. Is it ok if we leave it as is for now?

Reply:
Ok sounds good. I think we should eventually refactor that to make a standalone volume bar